app/models/concerns/ats/ashbyhq/xxx_application_fields.rb

```diff
@@ -0,0 +1,43 @@
+module Ats
```

Comment:
This is a template class presumably?

Reply:
I will delete it. It was a easy fix to have a backup of the previous version of that file.